### PR TITLE
Update Training Material Request #318

### PR DESCRIPTION
**Pull request by sachinkum0009**
Update Training Material Request #318
Added the screenshots of Ubuntu 18.04 and editing as per requirements.

### CODE REVIEW

**Comment by sachinkum0009** [comment]
added touch command and updated the document and ready for the pull request.

File: gh_pages/_source/prerequisites/Navigating-the-Ubuntu-GUI.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Things to notice:
 
 1. The close, minimize, and maximize buttons typically found on the right-hand side of the window title bar are found on the left-hand side.
 2. The menu for windows are found on the menu bar at the top of the screen, much in the same way Macs do. The menus, however, only show up when you hover the mouse over the menu bar.
-3. Notice that there are triangles on the left and right of the folder icon. The triangles on the left show how many windows of this application are open, and the right shows which application is currently in the foreground, or "has focus". Clicking on these icons when the applications are open does one of two things:
+3. Notice that there are menu hightlights of the folder icon. The dots on the left show how many windows of this application are open. Clicking on these icons when the applications are open does one of two things:
```
Comment (marip8) [inline gh_pages/_source/prerequisites/Navigating-the-Ubuntu-GUI.md]: Typo in `hightlights`; should be *highlights*

File: gh_pages/_source/prerequisites/The-Linux-Terminal.md [hunk 2]
```diff
@@ -111,6 +111,10 @@ _If you want a full list of options available for any of the commands given in t
  1. Type `mkdir new2`, then `ls`.
     * You will see there is a new folder `new2`.
 
+### touch Command ###
+ 1. Type `touch ~/Templates/"Untitled Document"`.
+    * This will create a new Document named **"Untitiled Document"**
```
Comment (marip8) [inline gh_pages/_source/prerequisites/The-Linux-Terminal.md]: Typo in `Untititled`; should be *Untitled*

**Review by marip8 — CHANGES_REQUESTED** [review]

**Comment by marip8** [comment]
@sachinkum0009 are the images you added in this PR used anywhere in the markdown documentation? If not, we should consider including them in the appropriate places in the documentation or removing them (unless there is a compelling reason to have them in the repository but not the documentation)

**Comment by sachinkum0009** [comment]
Yes, the images added are used in the markdown file. You can review the changes.

**Comment by marip8** [comment]
I built the documentation, and I can see the new images. From the PR, they looked like new (opposed to modified existing) images and I didn't see where they were used in the diff.

**Review by marip8 — APPROVED** [review]

**Comment by marip8** [comment]
@sachinkum0009 thanks for the contribution

**Comment by sachinkum0009** [comment]
@marip8 it is my pleasure.

**Pull request merged**